torch/csrc/jit/tensorexpr/tensor.h

```diff
@@ -391,6 +387,21 @@ inline ExprHandle Tensor::call(const std::vector<T>& args) {
   std::vector<ExprHandle> params(args.begin(), args.end());
   return FunctionCall::make(this, params);
 }
+
+template <typename... Ts>
+inline ExprHandle Placeholder::load(const Ts&... ts) const {
+  std::vector<ExprHandle> params({ExprHandle(ts)...});
+  return ExprHandle(
+      new Load(data(), ExprHandleVectorToExprVector(params), new IntImm(1)));
```

Comment:
Kind of OT and it's maybe inevitable but I always feel sad seeing these functions to convert-a-vector-of-one-thing-to-a-different-thing.  It feels like our APIs have internal impedance mismatches.  🤷

Reply:
That's what we get for having handles and raw pointers :) Good thing is that's it's usually hidden in not-so-frequently changed functions.